autopilot/betweenness_centrality.go

```diff
@@ -84,7 +84,7 @@ func NewBetweennessCentralityMetric(workers int) (*BetweennessCentrality, error)
 
 // Name returns the name of the metric.
 func (bc *BetweennessCentrality) Name() string {
-	return "betweeness_centrality"
```

Comment:
Since we use this name for automatically parsing config arguments for the various heuristics,  was wondering if changing it could break some configs. It looks like it is okay though, since the `NodeMetric` is only used internally. ping @bhandras to confirm

Reply:
This is a separate thing from the config. We don't really use the `NodeMetric.Name()`. (might as well remove it).